extension.js

```diff
@@ -22,55 +21,50 @@ const activate = async (context) => {
 	
 	if(response === 'Yes') {
 		vscode.window.showInformationMessage('Have a great day!');
-		intervalId = setInterval(() => {
+
+		intervalId = setInterval(async() => {
 			const lapTime = Date.now();
 			const incrementOfTime = Math.round((lapTime - startTime) / 60000);
+			const convertedTime = convertTime(incrementOfTime);
+			
+			// CHANGE TO HEROKU
+			const randomTip = await fetch.get('http://be-human-demo-staging.herokuapp.com/api/v1/tips/random');
+
+			const userChoice = await vscode.window.showInformationMessage(`You have been working for ${convertedTime}. Quick tip: ${randomTip.body.tip} Have time for a short break?`, 'Move your body', 'Not this time');
+
+			if (userChoice === 'Move your body') {
+				const randomLink = await fetch.get('http://be-human-demo-staging.herokuapp.com/api/v1/links/random')
+				vscode.env.openExternal(vscode.Uri.parse(`${randomLink.body.url}`));
+			}
+		}, 60000)
 
-			vscode.window.showInformationMessage(`You have been working for ${incrementOfTime} minutes.`, 'Disable');
-		}, timeIncrement)
 	} else if(response === 'No'){ 
 		vscode.window.showInformationMessage('Let\'s try again tomorrow!');
 	}
-	let disposable = vscode.commands.registerCommand('be-human.helloWorld', function () {
-		vscode.window.showInformationMessage('Welcome to beHuman!');
-	});
-
-//USED FOR TESTING ONLY????
-	// let trackTime = vscode.commands.registerCommand('be-human.trackTime', function () {
-
-
-		setInterval(() => {
-			const lapTime = new Date();
-			const incrementOfTime = Math.round((lapTime - startTime) / 60000);
-
-
-	    // THIS IS WHERE STOP WATCH USER COMMAND FUNCTIONS BEGIN
 
     let resetTimer = vscode.commands.registerCommand('be-human.resetTime', function () {
 		clearInterval(intervalId);
 
-		intervalId = setInterval(() => {
+		vscode.window.showInformationMessage('You have reset your time.');
+
+		intervalId = setInterval(async() => {
 			const lapTime = Date.now();
 			const incrementOfTime = Math.round((lapTime - startTime) / 60000);
-			vscode.window.showInformationMessage(`You have been working for ${incrementOfTime} minutes.`, 'Disable');
-		}, timeIncrement)
+			const convertedTime = convertTime(incrementOfTime);
+			
+			const randomTip = await fetch.get('http://be-human-demo-staging.herokuapp.com/api/v1/tips/random');
 
+			const userChoice = await vscode.window.showInformationMessage(`You have been working for ${convertedTime}. Quick tip: ${randomTip.body.tip} Have time for a short break?`, 'Move your body', 'Not this time');
 
-		 return resetTimer
-    });
-
-	    // THIS IS WHERE STOP WATCH USER COMMAND FUNCTIONS BEGIN
+			if (userChoice === 'Move your body') {
+				const randomLink = await fetch.get('http://be-human-demo-staging.herokuapp.com/api/v1/links/random')
+				vscode.env.openExternal(vscode.Uri.parse(`${randomLink.body.url}`));
+			}
+		}, 60000)
```

Comment:
I think 10min would actually be 600,000s
1s = 1000ms * 60s = 1min * 10min = 600,000